cfme_test.sh

```diff
@@ -0,0 +1,50 @@
+#!/bin/bash
+
+PYTHON_ENV_PATH="$HOME/test-yaml/bin"
```

Comment:
Just noticed - sorry, this is particular to your env - we'll need to solve this, or make it so the script takes args, or a config file, or make it so that you start the script whilst being in your virtualenv to begin with and it can pick up some of those things.